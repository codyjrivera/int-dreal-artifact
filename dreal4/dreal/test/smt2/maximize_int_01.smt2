;; https://github.com/dreal/dreal4/issues/210
(set-logic QF_NRA)
(declare-const i Int)
(assert (< i 100))
(assert (< 1 i))
(maximize i)
(check-sat)
(get-model)
(exit)
