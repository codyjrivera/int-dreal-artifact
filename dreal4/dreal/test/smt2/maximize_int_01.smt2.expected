delta-sat with delta = 0.001
(model
  (define-fun i () Int 99)
)
